azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub_mqtt.py

```diff
@@ -48,8 +51,14 @@ def _run_op(self, op):
             else:
                 client_id = op.device_id
 
-            username = "{hostname}/{client_id}/?api-version=2018-06-30".format(
-                hostname=op.hostname, client_id=client_id
+            query_params = [
```

Comment:
>query_params [](start = 12, length = 12)

I would like to rename this variable to something different that gives the idea that we have query_param kind of structure in an username field but actually they are not true query_params